### DISCUSSION TITLE
Extensions emit C(R)UD actions

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#22095] [Feature Requests]
### Summary

In order for other extensions to listen for extensions changes they should emit create update and delete actions like other collections.

### Motivation

Listen to extension changes and require an extension from another extension.

### Detailed Design

Same as in other services

### Must Have:
- Emit actions for create, update and delete events

### Drawbacks

n/A

### Alternatives

n/a

### Adoption Strategy

n/a

**Comment by JayShoe** [comment]
Good idea, Adding read emit action could be useful. IE "User downloaded this file", or "user viewed this profile".
